PointGeneration.cs

```diff
@@ -0,0 +1,6 @@
+using System;
+
+namespace Theframeworkshopskeleton
+{
+  

```

Comment:
//Step 1 : create rectangle around model
  //Step 2 : find percentage of 'air' in and out of the model's mesh
  //Step 3 : find the corresponding number of points to generate
  //Step 4 : generate and place all points (with all points in the binary tree)
  //Step 5 : remove all points that are out of the model's mesh
  //Step 6 : reposition all points (in while loop)
  //Step 7 : find straight lines, and remove unnecessary points
  //Step 8 : link all points together